tests/bench_timers/Makefile

```diff
@@ -8,6 +8,11 @@ SINGLE_TIMER_BOARDS = \
   native \
   nucleo-f031k6 \
   nucleo-f042k6 \
+  samd21-xpro \
```

Comment:
samd21-xpro has two timers declared, unless we have an issue here you can remove it.

Reply:
Done!